src/transformers/training_args.py

```diff
@@ -507,6 +507,12 @@ class TrainingArguments:
     _n_gpu: int = field(init=False, repr=False, default=-1)
 
     def __post_init__(self):
+        # expand paths, see #10628
```

Comment:
A link on the full issue would be better, or a short description even better.

Reply:
Better now ?

Reply:
Yes, make sure to run `make style` on your branch to fix the styling issue and we should be good to merge!

Reply:
this gives a lot more diff than expected !
see https://github.com/PaulLerner/transformers/commit/e7fc1eb55c4a3ee5189b688a7011d9efaf099336

Reply:
Are you sure you have the right versions pinned (normally `pip install -e .[dev]` will install those for you). Anyhow can push the styling on your branch if it doesn't work.

Reply:
right I've only installed the pip release, duly noted for a future PR :)